test/playwright/e2e/search-types.spec.ts

```diff
@@ -23,34 +23,38 @@ test.beforeEach(async ({ context }) => {
   await context.route('**.jamendo.com**', (route) => route.abort())
 })
 
-const searchTypes = [
-  {
-    id: 'all',
-    name: 'All content',
-    url: '/search/?q=birds',
-    canLoadMore: true,
-    metaSourceCount: 7,
-  },
-  {
-    id: 'image',
-    name: 'Images',
-    url: '/search/image?q=birds',
-    canLoadMore: true,
-    metaSourceCount: 7,
-    results: /Over 10,000 results/,
-  },
-  {
-    id: 'audio',
-    name: 'Audio',
-    url: '/search/audio?q=birds',
-    canLoadMore: true,
-    metaSourceCount: 3,
-    results: /93 results/,
-  },
-]
-
-async function checkLoadMore(page, searchType) {
-  const loadMoreSection = await page.locator('[data-testid="load-more"]')
+const allContentConfig = {
+  id: 'all',
+  name: 'All content',
+  url: '/search/?q=birds',
+  canLoadMore: true,
+  metaSourceCount: 7,
+} as const
+
+const imageConfig = {
+  id: 'image',
+  name: 'Images',
+  url: '/search/image?q=birds',
+  canLoadMore: true,
+  metaSourceCount: 7,
+  results: /Over 10,000 results/,
+} as const
+
+const audioConfig = {
+  id: 'audio',
+  name: 'Audio',
+  url: '/search/audio?q=birds',
+  canLoadMore: true,
+  metaSourceCount: 3,
+  results: /93 results/,
```

Comment:
These numbers make the tests flaky, but I'm not sure how to fix it. With all the hard work on the Catalog, there now are many more than 93 results for this search :)

Reply:
Shouldn't the tapes take care of that?